scripts/ts_dump.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# This script is used for backing up a single hypertable into an easy-to-restore
+# tarball. The tarball contains two files: (1) a .sql file for recreating the
+# hypertable and its indices and (2) a .csv file containing the data as CSV.
+#
+# Because pg_dump/pg_restore dump all of TimescaleDB's internal tables when
+# used, this script is useful if you want a backup that can be restored
+# regardless of TimescaleDB version running, or as part of a process where you
+# do not want to always backup all your hypertables at once.
+
+HYPERTABLE=${HYPERTABLE:-$1}
+PREFIX=${PREFIX:-$2}
```

Comment:
Why do this when you have to have the args on the command line as well for it to work properly ? I am also a bit worried about using environment variables with a risk of collision like PREFIX.

Reply:
I guess I was cargo culting a bit since we do something similar for most of our scripts. I can remove them as environmental vars.